libraries/APM_OBC/APM_OBC.cpp

```diff
@@ -117,6 +117,30 @@ const AP_Param::GroupInfo APM_OBC::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("MAX_COM_LOSS", 14, APM_OBC, _max_comms_loss, 0),
 
+    // @Param: GEOFENCE
+    // @DisplayName: Enable geofence Advanced Failsafe
+    // @Description: This enables the geofence part of the AFS. Will only be in effect if AFS_ENABLE is also 1
+    // @User: Advanced
+    AP_GROUPINFO("GEOFENCE",     15, APM_OBC, _enable_geofence_fs, 1),
```

Comment:
So to be clear, my thinking was that previously, if a user switched AFS_ENABLE to 1, all of those things (dual loss termination, RC termination, and geofence termination) would be ON by default, because there was previously no option to turn them off. When making this, I was thinking that it would be best to keep those behaviors the same - if someone switched AFS_ENABLE to 1, then those 3 termination conditions would be on by default. This way, if someone reflashes their firmware with the same parameters after this commit, there wouldn't be any unexpected changes to the termination behavior.

It makes sense that conditions that would cause the aircraft to crash would be disabled by default whenever possible though, I'm just trying to make sure I understand why the default values are the way they.